### ISSUE TITLE
A11y_AzureToolsStorageExplorer_BlobContainer_BlobContainerAcquireLease_ScreenReader: Title is not provided to the Acquire Lease dialog.

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#1976]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1;#MAS2.4.2 #VoiceOver; #Mac; 
**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
MAC Version 10.14.6 Beta(18G29g)
**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Right click on created blob container and select "Acquire Lease" control and hit enter.
6. Start VoiceOver and check whether the Voiceover reads the title of the dialog as Acquire Lease when opened.
**​Actual:​**
Voiceover does not read the title of the dialog as Acquire Lease dialog is opened after click on Acquire lease menu.
**Expected:**
VoiceOver should read the title as Acquire Lease dialog opened.
UserImpact:
Screen reader users unable to know the which dialog opened if it will not read the title.
**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:**
MAS1.3.1:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
MAS2.4.2:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={c74ff881-82b6-4401-92cf-8919f9bffaf2}
**Attachment for Reference:**
[1976_A11y_AzureToolsStorageExplorer_BlobContainer_BlobContainerAcquireLease_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3669221/1976_A11y_AzureToolsStorageExplorer_BlobContainer_BlobContainerAcquireLease_ScreenReader.pptx)

**Comment by MRayermannMSFT** [comment]
Fix checking into master.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT Verified on version 1.12.0 and MAC 10.15.3.The issue is no longer reproduced.Hence close this bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
